blog/relacja-z-meetupu-hitachi-karate/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: 'Relacja z meetupu Hitachi Energy - edycja Karate'
+authors: niewiarowski
+date: '2023-09-23'
+tags:
+  - 'news'
+  - 'z-kraju'
+  - 'meetcontent'
+  - 'spolecznosc'
+coverImage: '1693820333064.png'
+---
+
+Jako ekipa techwriterów i techwriterek z [Hitachi Energy](https://www.linkedin.com/company/hitachienergy/mycompany/) w składzie: Julia Dąbrowska, Mateusz Barszcz, Piotr Sroka, Monika Dąbrowska, Andrzej Niewiarowski oraz członkini honorowa - Patrycja Pyrek - przeprowadziliśmy wyjątkowy meetup, którego spoiwem była kultura Japonii. 
+
+<!--truncate-->
+
+## Fusion Meetup o Technical Writingu i Karate
+### Informacje o spotkaniu
+
+![](images/meta_prez.png)
+
+12.09.2023 w siedzibie Hitachi Energy w Krakowie odbył się meetup dla pracowników firmy i gości z poza niej. Na spotkaniu stawiło się 40 osób. Piotr wziął na siebie dużą część obowiązków organizacyjnych, w czym ma już dużą wprawę, bo ma już na koncie przeprowadzenie writerskiego meetupu w 2019. Dla naszego zespołu ten fusion meetup to już drugie wspólne wydarzenie, bo w obecnym roku Anno Domini 2023 zorganizowaliśmy spotkanie z [Lance Cummings](https://www.linkedin.com/in/lance-cummings-phd/), który w krakowskiej siedzibie firmy opowiadał o tym, jak wykorzystywać *prompt engingeering* w arsenale pisarza technicznego.
```

Comment:
```suggestion
12.09.2023 w siedzibie Hitachi Energy w Krakowie odbył się meetup dla pracowników firmy i gości spoza niej. Na spotkaniu stawiło się 40 osób. Piotr wziął na siebie dużą część obowiązków organizacyjnych, w czym ma już dużą wprawę, bo ma już na koncie przeprowadzenie writerskiego meetupu w 2019. Dla naszego zespołu ten fusion meetup to już drugie wspólne wydarzenie, bo w obecnym roku Anno Domini 2023 zorganizowaliśmy spotkanie z [Lance Cummings](https://www.linkedin.com/in/lance-cummings-phd/), który w krakowskiej siedzibie firmy opowiadał o tym, jak wykorzystywać *prompt engingeering* w arsenale pisarza technicznego.
```

Reply:
Jeżeli każdy akapit jest jedną linią, dodawanie komentarzy będzie zawsze dodaniem komentarza do całego akapitu. W Markdownie można stworzyć akapit pisząc po sobie wiele linii i oddzielamy go od następnego akapitu pusta linią.

W naszym repo mamy skonfigurowany Prettier. Jeżeli masz zainstalowany dodatek Prettier Code Formatter w VS Code, to możesz sformatować dokument naciskając Ctrl+S (Cmd+S na Macu). Wtedy każdy akapit podzieli się na linie o długości do 80 znaków i będzie łatwiej zrozumieć czego tyczy się dany komentarz.

Reply:
ok, zrobie tak nastepnym razem.